com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Math/Advanced/PosterizeNode.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class PosterizeNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,     // Version
+            "Posterize", // Name
```

Comment:
Maybe we want to get rid of the comments here?

Reply:
I like the comments because there's no other indication what these items are without them.